src/Components/Components/src/Forms/InputComponents/InputPassword.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.RenderTree;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    /// <summary>
+    /// An input component for editing password strings.
+    /// </summary>
+    public class InputPassword : InputText
+    {
+        /// <inheritdoc />
+        protected override void BuildRenderTree(RenderTreeBuilder builder)
+        {
+            builder.OpenElement(0, "input");
+            builder.AddAttribute(1, "id", Id);
+            builder.AddAttribute(2, "class", CssClass);
+            builder.AddAttribute(3, "value", BindMethods.GetValue(CurrentValue));
+            builder.AddAttribute(4, "onchange", BindMethods.SetValueHandler(__value => CurrentValue = __value, CurrentValue));
```

Comment:
Should this be `EventCallbackFactory`?

Reply:
As part of the API review I'd like to audit all of the things the compiler users. We still have `BindMethods` sitting around, but I think it only gets used now for the `GetValue` stuff.

Reply:
The code here is identical to the equivalent in `InputText`. If `EventCallbackFactory` is required, we should update all the forms components. If so let's do that separately from this PR.

Reply:
sure!